Kudu.Contracts/SiteExtensions/SiteExtensionInfo.cs

```diff
@@ -40,6 +40,7 @@ public SiteExtensionInfo(SiteExtensionInfo info)
             LocalIsLatestVersion = info.LocalIsLatestVersion;
             LocalPath = info.LocalPath;
             InstalledDateTime = info.InstalledDateTime;
+            InstallationArgs = info.InstallationArgs;
```

Comment:
Keep everything as is. It's the ARM idempotent behavior. Doing the same PUT must be a no-op.

Of course in this case it's not quite the same....

To solve it, we'd need to:

- store the args somewhere as part of installation
- Check that it matches what it used to be, and fail if not

Either that, or treat as a re-install if version is same but args don't match.